googlehttpclient/src/main/java/feign/googlehttpclient/GoogleHttpClient.java

```diff
@@ -112,6 +114,7 @@ private final Response convertResponse(final Request inputRequest,
       contentLength = inputResponse.getHeaders().getContentLength().intValue();
     }
     return Response.builder()
+        .protocolVersion(HTTP_PROTOCOL_VERSION)
```

Comment:
remove, let Response default value deal with this one.

Reply:
done